artemis-commons/src/main/java/org/apache/activemq/artemis/utils/critical/CriticalAnalyzerImpl.java

```diff
@@ -147,6 +147,7 @@ public void run() {
                      // this means that the server has been stopped as we could acquire the semaphore... returning now
                      break;
                   }
+                  logger.trace("Checking critical analyzer");
```

Comment:
Was this left in by mistake?  I ask because it looks like it will be on the critical path for any checks.  Maybe surround it with if (traceEnabled)?

Reply:
nope.. not by mistake...

I talked to the guy who wrote jboss-logging once... the issue that we would need if (logger.istTraceEnabled()) was for concatenations... this is a constant string.. and has no cost from a previous conversation...


I can add the if (log.isTraceEnablled)... but it won't buy is much


Besides.. this only happens every 2 minutes only.

Reply:
Got it.  No worries, then.  Leave it as-is.